File: JavaScript/Orden_filas_2D.js
const matriz2D = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

function ordenPorFilas(m) {
    let resultado = "";
    for (let i = 0; i < m.length; i++)
        for (let j = 0; j < m[i].length; j++)
            resultado += m[i][j] + " ";
    console.log(resultado);
}

ordenPorFilas(matriz2D);

File: Python/Orden_filas_2D.py
matriz2D = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

def orden_por_filas(m):
    for i in range(len(m)):
        for j in range(len(m[i])):
            print(m[i][j], end=" ")
    print()

orden_por_filas(matriz2D)